libs/api/domains/cms/src/lib/search/importers/article.service.ts

```diff
@@ -104,6 +104,11 @@ export class ArticleSyncService implements CmsSyncProvider<IArticle> {
                 value: x.title,
                 type: 'category',
               })),
+              ...(entry.fields?.organization ?? []).map((x) => ({
+                key: x.fields?.slug,
```

Comment:
Key is not optional, we should provide a default.
```suggestion
                key: x.fields?.slug ?? '',
```